screeps-profiler.js

```diff
@@ -147,7 +147,8 @@ const Profiler = {
       return 'Profiler not active.';
     }
 
-    const elapsedTicks = Game.time - Memory.profiler.enabledTick + 1;
+    const elapsedTicks = (Memory.profiler.type === 'background' ? Game.time :
```

Comment:
Can you split this line up? Maybe a `const endTick = ...` the two line ternary isn't terribly easy to read.